index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="form-title">
+      <h1 id="title">Survey Form</h1>
+      <p id="description">This is our first survey form project, please fill out.</p>
+    </div>
+    <form id="survey-form" action="#">
+      <div class="">
+        <label class="help-label" for="name" id="name-label">Name: </label>
+        <input type="text" id="name" name="name" required placeholder="Input your name ">
+      </div>
+      <div class="">
+        <label class="help-label" for="email" id="email-label">Email: </label>
+        <input type="email" id="email" name="email" required placeholder="Input your emal adress ">
+      </div>
+      <div class="">
+        <label class="help-label" for="number" id="number-label">Age: </label>
+        <input type="number" id="number" name="number" min="1" max="110" required placeholder="Input your age">
+      </div>
+      <div class="">
+        <label class="help-label" for="dropdown">Occupation:</label>
+        <select id="dropdown" name="dropdown" class="form-control" required="" >
+          <option value="">Select current role</option>
+          <option value="student">Be an excelent student</option>
+          <option value="full-time-jon">Full time job</option>
+          <option value="full-time-learner">Ful time learner</option>
+        </select>
+      </div>
+      <div class="">
```

Comment:
Your HTML is really well laid out. We can make it even better.
[OPTIONAL] I notice you insert class attribute without value. In order to have a clean markup code, you can remove it for now. You can add these attributes later if you need.

Reply:
Hi @gabyse1, This is great but I think it shouldn't be optional. the class attribute should be removed if it has no value

Reply:
Your HTML is really well laid out. We can make it even better.
I notice you insert class attribute without value. In order to have a clean markup code, you can remove it for now. You can add these attributes later if you need.